arwenmandos/util.go

```diff
@@ -155,6 +158,45 @@ func addESDTToVMInput(esdtData []*mj.ESDTTxData, vmInput *vmcommon.VMInput) {
 	}
 }
 
+// CreateMultiTransferData builds data for a multiTransferESDT
+func CreateMultiTransferData(to []byte, esdtData []*mj.ESDTTxData, endpointName string, arguments [][]byte) []byte {
```

Comment:
also move this entire file and the tests to mandos-go/util, other projects will also make use of it